e2e/benchmarks/browserstack-benchmark/package.json

```diff
@@ -29,7 +29,7 @@
   "scripts": {
     "test": "karma start",
     "test-node": "jasmine app_node_test.js",
-    "build-deps": "yarn build-core && yarn build-backend-cpu && yarn build-backend-wasm && yarn build-backend-webgl && yarn build-converter && yarn build-layers && yarn build-data && yarn build-tfjs && yarn build-vis",
+    "build-all-deps": "yarn build-core && yarn build-backend-cpu && yarn build-backend-wasm && yarn build-backend-webgl && yarn build-converter && yarn build-layers && yarn build-data && yarn build-tfjs && yarn build-vis",
```

Comment:
This can be simplified: https://github.com/mattsoulanille/tfjs/blob/fix_browserstack_benchmarks/e2e/benchmarks/browserstack-benchmark/package.json#L32-L35

Reply:
Updated. Could `build-link-package` build individual package? In other words, if I only need local build for webgl, could I use `build-link-package` command? Or I still have to keep `build-backend-webgl` here?